public/editor/scripts/editor/js/fc/filesystem-sync.js

```diff
@@ -36,8 +36,8 @@ define(function(require) {
 
     // Update the UI with a "Saving..." indicator whenever we sync a file
     syncManager.on("file-sync-start", function() {
+	  $("#navbar-save-indicator").removeClass("hide");
       $("#navbar-save-indicator").text("{{ fileSavingIndicator }}");
```

Comment:
tabs. You have a tab *and* spaces on this line, so do a regexp replace to turn `\t` into `    ` (four spaces) in this file.